common-libs/common/src/shared/DidResolver.ts

```diff
@@ -1,13 +1,19 @@
 import { RegistryApiService } from '@affinidi/internal-api-clients'
 import { DidDocument } from './interfaces'
+import LRUCache from 'lru-cache'
```

Comment:
let's order imports in that way

external libs
external affinidi libs
internal imports